README.md

```diff
@@ -1,3 +1,10 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+1. Install npm init -y 
+2. Install jest 
+3. Install husky 
+4. Install pg to run database 
+5. Install express to set up server 
+6. npm run setup-dbs to create database 
+7. npm install dotenv --save
+8. npm run seed to add data to database 
```

Comment:
You can use backticks to make `text all codey`
I would include instructions here about making the .env files and what needs to go in them